styx-common/src/main/java/com/spotify/styx/util/ShardedCounter.java

```diff
@@ -246,8 +223,8 @@ public long getCounter(String counterId) {
     final CounterSnapshot snapshot = getCounterSnapshot(counterId);
     long result = 0;
 
-    for (long shard : snapshot.shards.values()) {
-      result += shard;
+    for (long shardValue : snapshot.getShards().values()) {
```

Comment:
Let's do it like `snapshot.getShards().values().stream().mapToLong(i -> i).sum()`